### ISSUE TITLE
IMPORTANT: 1.0.0 Release 🎉

### ISSUE THREAD

**Issue by Alexays** [Alexays/Waybar#650]
Hi!
I've been pretty busy lately, but I'm going to get back into Waybar and start working on the 1.0.0 release.
Here the [1.0.0 milestone](https://github.com/Alexays/Waybar/milestone/5).

I'm gonna go around all the issues and put some in this milestone.
But I also wanted to ask you which ones you find priority and want to see soon!
If you have ideas for improvement, don't hesitate!
And don't hesitate to add your own modifications!
Can't wait to share this with you!

Alex.

**Comment by BlueGone** [comment]
First of all, congratulations on the progress of the project. It has simply become great, and de facto, a great standard. :tada:
Then, for the 1.0.0 milestone, I'm interested in issue #66 . I've seen the work done on PR #556 , and I think I can fork it and fix the problems that have been raised. Let me know if it's a feature that fits in the milestone.

**Comment by maximbaz** [comment]
Great progress on the project @Alexays 🎉 

There two issues hurt me most: #388 (already part of the milestone) and at the end of every month #581. I would be happy to help with #581, if you could comment in that issue your thoughts of the preferred approach?

**Comment by ZJaume** [comment]
#434 would be great, I think configuration file has to be more easy and confortable to edit, I think YAML or TOML are the way to go.

**Comment by yory8** [comment]
To me it's #255 , which seems to have a fix in this fork https://github.com/xPMo/Waybar/commit/8e6b6a781dc4b1fdd53eca57cf0d21620dfba59c

**Comment by Alexays** [comment]
@yory8 @xPMo any news about #255 ? 😃

**Comment by yory8** [comment]
@Alexays i'm not involved with that effort; it seems they got stuck with some segfaults?

**Comment by bluebyt65** [comment]
I like to have a current keyboard layout but not based on sway, some people use waybar with wayfire.

**Comment by luispabon** [comment]
https://github.com/Alexays/Waybar/issues/509 and https://github.com/Alexays/Waybar/issues/585 would be at the top of my list and seem to be related and/or the same issue?

**Comment by Alexays** [comment]
> #509 and #585 would be at the top of my list and seem to be related and/or the same issue?

This one should be fixed, can you test?

**Comment by bluebyt65** [comment]
I feature nice to have a "taskbar/window list" like this: 
https://www.reddit.com/r/Polybar/comments/gsxdh3/polywins_a_customizable_polybar_taskbarwindow_list/

**Comment by luispabon** [comment]
> > #509 and #585 would be at the top of my list and seem to be related and/or the same issue?
> 
> This one should be fixed, can you test?

They seem to be, :ok_hand:

**Comment by tinywrkb** [comment]
Flatpak package? [This is my Flatpak manifest](https://github.com/tinywrkb/flatpaks/blob/master/com.github.alexays.waybar/com.github.alexays.waybar.yaml) and it's how I'm now runnning Waybar.  
So far I didn't notice issues with this but my needs from Waybar are very basic.  
Note that the manifest needs some cleanup and I packaged in Sway for swaymsg and also jq.

**Comment by WhyNotHugo** [comment]
@tinywrkb Scaling does not seem to work via flatpak, so that should probably be addressed before encouraging users to adopt that.

**Comment by tinywrkb** [comment]
@WhyNotHugo [this](https://github.com/flatpak/flatpak/issues/3619)? sounds like a GTK issue, not a Wayland one.  
What do you mean by encouraging users? it just a distribution method, users should be free to choose how to install apps. The fact that some issue is affecting your setup doesn't mean Flatpak packaging should not be considered, it's up to the developer to decide.  
Note that it's possible to add a disclaimer to the Flathub app page about the issue or to keep the app on Flathub-beta repo until the issue is resolved.

**Comment by WhyNotHugo** [comment]
> sounds like a GTK issue, not a Wayland one.

Not entirely certain where the fault lies -- applications scale fine when running natively on Wayland, it's just via flatpak that they fail. I can't honestly figure out if the issue is Wayland, Flatpak, or GTK.

**Comment by tinywrkb** [comment]
@WhyNotHugo do you have gnome-settings-daemon installed? see if the xsettings schema helps. If you're on Arch Linux then I already packaged the xsettings schema [here](https://aur.archlinux.org/packages/gsettings-xsettings-schemas/).  
I don't know about scaling but it helps with having the fonts rendered correctly and it suppose to be moved from gnome-settings-daemon into gsettings-desktop-schemas.

**Comment by eoli3n** [comment]
any news?

**Comment by WhyNotHugo** [comment]
No, I don't have `gnome-settings-daemon`. The [README](https://github.com/GNOME/gnome-settings-daemon) indicate that this is a GNOME-specific thing:

> The GNOME Settings Daemon is responsible for setting various parameters of a
> GNOME Session and the applications that run under it.

I'm using swaywm.

I did install it anyway. Apps now draw a different kind of blurry. Fonts are very blurry, but images are sharp. So it's no longer a scaling problem, but some other rendering problem...?

**Comment by eoli3n** [comment]
@Alexays i don't see https://github.com/Alexays/Waybar/issues/434 in the milestone

**Comment by cinerea0** [comment]
Would you consider making another intermediate release between now and 1.0? There were some commits merged back in October dealing with the taskbar module that would be great to have in a tagged release.

**Comment by WillPower3309** [comment]
> I feature nice to have a "taskbar/window list" like this:
> https://www.reddit.com/r/Polybar/comments/gsxdh3/polywins_a_customizable_polybar_taskbarwindow_list/

This could be super useful for also keeping track of scratchpad windows

**Comment by jsteinaker** [comment]
Hi, thanks for your great work. I definitely think #917 should be on the list, there must be a way to
a) dynamically set taskbar's max width, based on the modules present (it should be able to use as much space as it wants, as long as it leaves space for the modules remaining)
b) set each button's max width, if I open a webpage with a very long title in Firefox, that button takes a LOT of space.

**Comment by WuerfelDev** [comment]
A suggestion from #1349 which would be cool to see implemented is a possibility to set/remove a css class on `window#waybar` via ipc
